tests/e2e/networking/00-asserts.yaml

```diff
@@ -0,0 +1,134 @@
+apiVersion: networking.k8s.io/v1
+kind: NetworkPolicy
+metadata:
+  labels:
+    app.kubernetes.io/managed-by: operator-lifecycle-manager
+    app.kubernetes.io/name: tempo-operator
+    app.kubernetes.io/part-of: tempo-operator
+    control-plane: controller-manager
+  name: tempo-operator-egress-to-apiserver
+  namespace: ($TEMPO_NAMESPACE)
+spec:
+  egress:
+  - ports:
+    - port: 6443
+      protocol: TCP
+  podSelector:
+    matchLabels:
+      app.kubernetes.io/managed-by: operator-lifecycle-manager
+      app.kubernetes.io/name: tempo-operator
+      app.kubernetes.io/part-of: tempo-operator
+      control-plane: controller-manager
+  policyTypes:
+  - Egress
+---
+apiVersion: networking.k8s.io/v1
+kind: NetworkPolicy
+metadata:
+  labels:
+    app.kubernetes.io/managed-by: operator-lifecycle-manager
+    app.kubernetes.io/name: tempo-operator
+    app.kubernetes.io/part-of: tempo-operator
+    control-plane: controller-manager
+  name: tempo-operator-deny-all
+  namespace: ($TEMPO_NAMESPACE)
+spec:
+  podSelector:
+    matchLabels:
+      app.kubernetes.io/managed-by: operator-lifecycle-manager
+      app.kubernetes.io/name: tempo-operator
+      app.kubernetes.io/part-of: tempo-operator
+      control-plane: controller-manager
+  policyTypes:
+  - Ingress
+  - Egress
+---
+apiVersion: networking.k8s.io/v1
+kind: NetworkPolicy
+metadata:
+  labels:
+    app.kubernetes.io/managed-by: operator-lifecycle-manager
+    app.kubernetes.io/name: tempo-operator
+    app.kubernetes.io/part-of: tempo-operator
+    control-plane: controller-manager
+  name: tempo-operator-ingress-to-metrics
+  namespace: ($TEMPO_NAMESPACE)
+spec:
+  ingress:
+  - from:
+    - namespaceSelector: {}
+      podSelector: {}
+    ports:
+    - port: metrics
```

Comment:
I have realized that if port name is used instead of a number ingress access to all ports is allowed.

Reply:
Is this documented somewhere?  :octocat:

Reply:
I couldn't find it, but I didn't find examples of using port names.

Reply:
@pavolloffay when I remove the webhook policy, allowing incoming traffic on port `9443` and maintain the `%s-ingress-to-metrics` policy using the port named `metrics` I am unable to access `9443`.

```
$ oc version                                                                                                                    
Client Version: 4.17.1
Kustomize Version: v5.0.4-0.20230601165947-6ce0bf390ce3
Server Version: 4.19.4
Kubernetes Version: v1.32.6
```